lib/mayaUsd/render/vp2RenderDelegate/instancer.cpp

```diff
@@ -77,12 +82,18 @@ void HdVP2Instancer::_SyncPrimvars()
 
     // Use the double-checked locking pattern to check if this instancer's
     // primvars are dirty.
-    int dirtyBits = changeTracker.GetInstancerDirtyBits(id);
-    if (HdChangeTracker::IsAnyPrimvarDirty(dirtyBits, id)) {
+    HdDirtyBits dirtyBits = changeTracker.GetInstancerDirtyBits(id);
+    if (HdChangeTracker::IsAnyPrimvarDirty(dirtyBits, id)
+        || HdChangeTracker::IsInstancerDirty(dirtyBits, id)) {
         std::lock_guard<std::mutex> lock(_instanceLock);
 
         // If not dirty, then another thread did the job
         dirtyBits = changeTracker.GetInstancerDirtyBits(id);
+
+#if defined(HD_API_VERSION) && HD_API_VERSION >= 36
+        _UpdateInstancer(GetDelegate(), &dirtyBits);
```

Comment:
It looks like this should only be done when the instancer is dirty?

Reply:
Yup, and the base class implementation in `HdInstancer` checks that:
https://github.com/PixarAnimationStudios/USD/blob/f7b5f25e0d7bd951439485841304aa430683ab23/pxr/imaging/hd/instancer.cpp#L119